cmd/atlas/cmd/issues_test.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"flag"
+	"testing"
+
+	"github.com/urfave/cli"
+)
+
+const (
+	testRepo    = "https://github.com/ahstn/atlas"
+	expectedURL = "https://github.com/ahstn/atlas/issues"
+)
+
+func TestDetermineURLUsingFlag(t *testing.T) {
+	set := flag.NewFlagSet("test", 0)
+	set.String("url", testRepo, "usage")
+	c := cli.NewContext(nil, set, nil)
+
+	url, err := determineURL(c)
+	if err != nil {
+		t.Fatal("Failed to determine URL:", err)
+	}
+
+	if url != expectedURL {
+		t.Fatalf("Expected determineURL to return '%s'. Got: %v", expectedURL, err)
+	}
+}
+
+// Even though this relies on Git, I'm happy keeping it as the remote URL should
+// always be the GitHub URL for ahstn/atlas
```

Comment:
Silly assumption, looks like the CircleCI container doesn't have Git installed.

So for now this should probs be commented out and re-enabled when I add Git into the build agent container.
- [ ] Disable `TestDetermineURLUsingGit()` for now.